src/utils/functionsSort.js

```diff
@@ -11,3 +11,16 @@ export function sortByMonthAndYear(a, b) {
     return a.annee < b.annee ? -1 : 1;
   }
 }
+
+export function sortSousThemes(sousThemes) {
+  return sousThemes.sort(function compare(a, b) {
```

Comment:
ok on pourrait le passer en générique ?
```suggestion
export function sortByString(array) {
  return array.sort(function compare(a, b) {
```